src/main/java/org/alliancegenome/curation_api/jobs/executors/VariantExecutor.java

```diff
@@ -19,42 +19,45 @@
 @ApplicationScoped
 public class VariantExecutor extends LoadFileExecutor {
 
-	@Inject
-	VariantDAO variantDAO;
-	@Inject
-	VariantService variantService;
+	@Inject VariantDAO variantDAO;
+	@Inject VariantService variantService;
 
 	public void execLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 
 		BulkManualLoad manual = (BulkManualLoad) bulkLoadFile.getBulkLoad();
 		Log.info("Running with: " + manual.getDataProvider().name());
 
 		IngestDTO ingestDto = readIngestFile(bulkLoadFile, VariantDTO.class);
-		if (ingestDto == null) return;
-		
+		if (ingestDto == null) {
+			return;
+		}
+
 		List<VariantDTO> variants = ingestDto.getVariantIngestSet();
-		if (variants == null) variants = new ArrayList<>();
-		
+		if (variants == null) {
+			variants = new ArrayList<>();
+		}
+
 		BackendBulkDataProvider dataProvider = manual.getDataProvider();
-		
+
 		List<Long> variantIdsLoaded = new ArrayList<>();
 		List<Long> variantIdsBefore = new ArrayList<>();
 		if (cleanUp) {
 			variantIdsBefore.addAll(variantService.getIdsByDataProvider(dataProvider.name()));
 			Log.debug("runLoad: Before: total " + variantIdsBefore.size());
 		}
-		
+
 		bulkLoadFile.setRecordCount(variants.size() + bulkLoadFile.getRecordCount());
 		bulkLoadFileDAO.merge(bulkLoadFile);
-		
+
 		BulkLoadFileHistory history = new BulkLoadFileHistory(variants.size());
 		createHistory(history, bulkLoadFile);
 		boolean success = runLoad(variantService, history, dataProvider, variants, variantIdsLoaded);
-		if(success && cleanUp) runCleanup(variantService, history, bulkLoadFile, variantIdsBefore, variantIdsLoaded);
+		if(success && cleanUp) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.